solr/solrj/src/java/org/apache/solr/client/solrj/io/eval/ListCacheEvaluator.java

```diff
@@ -41,35 +39,21 @@ public ListCacheEvaluator(StreamExpression expression, StreamFactory factory) th
   }
 
   @Override
-  @SuppressWarnings({"unchecked"})
   public Object doWork(Object... values) throws IOException {
-    @SuppressWarnings({"rawtypes"})
-    ConcurrentMap objectCache = this.streamContext.getObjectCache();
-    @SuppressWarnings({"rawtypes"})
-    List list = new ArrayList();
+    Map<String, Object> objectCache = this.streamContext.getObjectCache();
```

Comment:
```suggestion
    ConcurrentMap<String, Object> objectCache = this.streamContext.getObjectCache();
```

This would help with code reading clarity I think.